spec/steps/prepare.fmf

```diff
@@ -22,6 +22,18 @@ description: |
             how: shell
             script: dnf install -y httpd
 
+/install:
+    summary:
+        Install a package using dnf/yum.
+    description:
+        Installs a package from specified source and verifies that it was
+        installed successfuly.
```

Comment:
So the plan for this method is to support installing from all these sources?

 * default enabled repositories
 * copr
 * koji/brew

If so, could you please include examples for each source?

Reply:
Yes, that's in the TODO. :)

Reply:
What are `default enabled repositories`?

Reply:
By `default enabled` I meant basically doing `dnf install -y $packages` (no `--enablerepo`).

Reply:
Ok, that's correct then.

Reply:
This does not seem to be resolved. There is still only the `copr` example. It's important to have at least one example for each method.

Reply:
It's on the TODO (see the description). Please don't make me track it in conversations.

Reply:
I see, I haven't realized this is a WIP.

Reply:
Sorry, I did not state it explicitly before. I'll make it `WIP` every time it is :) for new PRs.